### PR TITLE
edit and delete category

### PR DESCRIPTION
**Pull request by victoriadv**

### CODE REVIEW

**Review by volodymyr2000mandzyniak — APPROVED** [review]

**Review by volodymyr2000mandzyniak — APPROVED** [review]

**Comment by Luzhnuy** [comment]
Please merge your two branches and open one pull request

File: models/articles/models.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 
 class Article(db.Model):
+    __tablename__ = 'article'
```
Comment (Luzhnuy) [inline models/articles/models.py]: Please rename tables to be plural name.
For example:
article -> articles
category - > categories

File: routes/articles/routes.py [hunk 2]
```diff
@@ -71,9 +71,30 @@ def category_save():
...
+    return render_template("main/category_form.html", category=category)
+
+@app.route("/category/<int:id>/update", methods=["POST"])
+def category_update(id):
+    category = Category.query.get(id)
+    category.title = request.form.get("title")
+    db.session.add(category)
+    db.session.commit()
+    return redirect("/all")
+@app.route("/category/<int:id>/delete")
```
Comment (Luzhnuy) [inline routes/articles/routes.py:93]: Add blank lines before other route

**Review by Luzhnuy — CHANGES_REQUESTED** [review]

**Review by volodymyr2000mandzyniak — APPROVED** [review]

**Review by volodymyr2000mandzyniak — APPROVED** [review]

**Review by volodymyr2000mandzyniak — APPROVED** [review]

**Review by volodymyr2000mandzyniak — APPROVED** [review]

**Comment by Luzhnuy** [comment]
Please resolve the comments and conflicts

**Comment by Luzhnuy** [comment]
Remove second pull request

**Review by Luzhnuy — APPROVED** [review]

**Review by Luzhnuy — APPROVED** [review]

**Pull request merged**